src/components/routes/homepage/EventFormContainer.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState, forwardRef } from 'react';
+import moment from 'moment';
+import {
+  AppBar,
+  Dialog,
+  Fab,
+  IconButton,
+  Tooltip,
+  Toolbar,
+  Typography,
+  Slide,
+  makeStyles
+} from '@material-ui/core';
+import { Add as AddIcon, Close as CloseIcon } from '@material-ui/icons';
+import { useMutation } from '@apollo/react-hooks';
+import { GET_ALL_EVENTS, CREATE_EVENT } from '../../../gql/events';
+import { EventForm } from './EventForm';
+import { Modal } from './Modal';
+import { Loading } from '../../shared/Loading';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    position: 'fixed',
+    bottom: 20,
+    right: 20
+  },
+  appBar: {
+    backgroundColor: theme.palette.primary.main
+  },
+  title: {
+    marginLeft: theme.spacing(2),
+    flex: 1
+  }
+}));
+
+const Transition = forwardRef(function Transition(props, ref) {
+  return <Slide direction="up" ref={ref} {...props} />;
+});
+
+export const EventFormContainer = () => {
+  const [open, setOpen] = useState(false);
+  const [modal, setModal] = useState({
+    isOpen: false,
+    title: ''
+  });
+
+  const classes = useStyles();
+
+  const [createEvent, { loading, error }] = useMutation(CREATE_EVENT, {
+    refetchQueries: [{ query: GET_ALL_EVENTS }]
+  });
+
+  if (error) {
+    console.log(error);
+  }
+
+  const doCreateEvent = async ({
+    title,
+    venue,
+    start,
+    end,
+    description,
+    image_url: image_base64
+  }) => {
+    await createEvent({
```

Comment:
don't think we need `async/await` here because we're not returning anything out of this function
only use `async/await` when a `Promise` is returned
`.then` and `.catch` are good

Reply:
ohh yeahh didn't use `.then` and `.catch` before, will remove.